tests/testthat/_snaps/emissions_profile.md

```diff
@@ -181,3 +181,45 @@
       24     0
       
 
+# At company level, three matched products with different `co2_footprint`, one missing benchmark, and one unmatched product yield the expected output
+
+    Code
+      missing_benchmark
+    Output
+      # A tibble: 8 x 4
+        companies_id grouped_by       risk_category value
+        <chr>        <chr>            <chr>         <dbl>
+      1 a            isic_4digit      high            0.2
+      2 a            isic_4digit      medium          0.2
+      3 a            isic_4digit      low             0.2
+      4 a            isic_4digit      <NA>            0.4
```

Comment:
Here I explain the output affected by a missing benchmark. I focus on rows 1-4 but the same goes for rows 5-8.

A total `value` of `1` is spread across 5 products, so the `value` for each product is `1 / 5 = 0.2`.

Two products yield `NA` (one because of a missing benchmark and another one because of an unmatched product), so in the row (4) where `risk_category` is `NA` the `value` is `2 * 0.2 = 0.4`.

Each of the other rows (1-3) show a matched product with a non-missing benchmark, so each `value` is simply `0.2`.